credentialdigger/models/snippet_model/snippet_model.py

```diff
@@ -201,3 +202,20 @@ def _remove_initial_junk(self, snippet):
             r"^((\s*|\ *)\@\@.*\@\@(\s*|\ *)|(\s*|\ *)\+(\s*|\ *)|(\s*|\ *)\-(\s*|\ *)|(\s*|\ *))",
             "",
             snippet).strip()
+
+    def _assignment_expression(self, snippet):
+        """ Check if the snippet is an assignment or not.
+        """
+        assignment_symbols = ['=', ':=', '<-', ':']
+        for symbol in assignment_symbols:
+            findings = re.findall(f'(.*){symbol}(.*)', snippet)
```

Comment:
isn't it sufficient a piece of code like this?

```
for symbol in symbols:
    return True if symbol in snippet
return False
```

Reply:
> isn't it sufficient a piece of code like this?
> 
> ```
> for symbol in symbols:
>     return True if symbol in snippet
> return False
> ```

Yes, at first I wanted to extract the findings using ReGex but then I did not follow that path so I ended up only checking for the existence of the symbols. will replace the code.